controller/volunteer_controller.py

```diff
@@ -45,25 +60,23 @@ def change_volunteer_camp(volunteer: Volunteer, camp: Camp, is_admin: bool) -> V
     :param camp: the new camp
     :return:
     """
+    # A new menu class EditVolunteerMenu was built to list the items to be edited including camp in the new branch cli-edit-volunteer
     pass
 
 
 def deactivate_volunteer(volunteer: Volunteer) -> Volunteer:
-    """
-    Yunsy, Yingbo
-    """
-    pass
+    volunteer_called = volunteer
+    volunteer_called.account_activated = False
+    volunteer_called.save()
+    return volunteer_called
```

Comment:
I think that we do not need to reassign `volunteer` to `volunteer_called`
So just:
```python
def deactivate_volunteer(volunteer: Volunteer) -> Volunteer:
    volunteer.account_activated = False
    volunteer.save()
    return volunteer
```